package.json

```diff
@@ -90,7 +91,7 @@
     "applicationinsights": "0.21.0",
     "moment": "^2.17.1",
     "open": "^0.0.5",
-    "vscode-azureextensionui": "~0.12.0",
-    "vscode-extension-telemetry": "^0.0.15"
+    "vscode-azureextensionui": "~0.18.0",
+    "vscode-extension-telemetry": "0.0.22"
```

Comment:
Please update this to 0.1.0 instead. We recently learned of an issue where there are multiple powershell processes being spawned and stay stuck on older Windows versions when using version 1.0.5 of `applicationInsights` module.

This [bug](https://github.com/Microsoft/ApplicationInsights-node.js/issues/438) has been fixed in  version 1.0.6 of `applicationInsights` module which is being used by version 0.1.0 of `vscode-extension-telemtetry`

Therefore, please update the version of `vscode-extension-telemetry` dependency to 0.1.0